examples/dreamer/README.md

```diff
@@ -0,0 +1,7 @@
+# Dreamer example
+
+## Note:
+This example is not included in the benchmarked results of the current release. The intention is to include it in the
```

Comment:
can you add 
```suggestion
This example is not included in the benchmarked results of the current release (v0.3). The intention is to include it in the
```
I intend to look for all the v0.3 in the next release and address them. It's easier for us to keep track if we pinpoint precisely every part of the code that needs to be revised before the release